### PR TITLE
[OSDOCS-9038]:Add new force-delete cluster flag "--best-effort" to "Managing objects with the ROSA CLI" docs

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-9038
Link to docs preview:

https://69211--ocpdocs-pr.netlify.app/openshift-rosa/latest/cli_reference/rosa_cli/rosa-manage-objects-cli#rosa-delete-cluster_rosa-managing-objects-cli
QE review:
- [ ] QE has approved this change.

Additional information:

image

### CODE REVIEW

File: modules/rosa-delete-objects.adoc [hunk 1]
```diff
@@ -75,6 +75,9 @@ $ rosa delete cluster --cluster=<cluster_name> | <cluster_id> [arguments]
 
 |--watch
 |Watches the cluster uninstallation logs.
+
+|--best-effort
+|Skips steps in the cluster destruction chain that are known to cause the cluster deletion process to fail. This option should be used with care and may require manual cleanup of certain resources.
```
Comment (mburke5678) [inline modules/rosa-delete-objects.adoc:80]:
```suggestion
|Skips steps in the cluster destruction chain that are known to cause the cluster deletion process to fail. This option should be used with care and might require manual cleanup of certain resources.
```

Comment (mburke5678) [reply modules/rosa-delete-objects.adoc:80]: | cleanup of certain resources

How do I determine if manual cleanup is needed, and what needs cleaning up?

**Comment by mburke5678** [comment]
@mletalie A couple of things. Otherwise LGTM

File: modules/rosa-delete-objects.adoc (hunk 1 above)
Comment (mletalie) [reply modules/rosa-delete-objects.adoc:80]: > | cleanup of certain resources
> 
> How do I determine if manual cleanup is needed, and what needs cleaning up?

Good question, and one I have been trying to get an answer to. Let me check again.

**Comment by mletalie** [comment]
> @mletalie A couple of things. Which version(s) to cherrypick to? Otherwise LGTM

Good catch. Updated info.

**Comment by mletalie** [comment]
Hello @mburke5678,
    Updated cherrypick info and made requested change.  Thx.

File: modules/rosa-delete-objects.adoc [hunk 2]
```diff
@@ -75,6 +75,9 @@ $ rosa delete cluster --cluster=<cluster_name> | <cluster_id> [arguments]
 
 |--watch
 |Watches the cluster uninstallation logs.
+
+|--best-effort
+|Skips steps in the cluster destruction chain that are known to cause the cluster deletion process to fail. You should use this option with care and it is recommended that you manually check your AWS account for  any resources that may be left over after using `--best-effort`.
```
Comment (mburke5678) [inline modules/rosa-delete-objects.adoc:80]:
```suggestion
|Skips steps in the cluster destruction chain that are known to cause the cluster deletion process to fail. You should use this option with care and it is recommended that you manually check your AWS account for any resources that might be left over after using `--best-effort`.
```

**Comment by adellape** [comment]
@mletalie I think @mburke5678's suggestion about "may" -> "might" should be addressed before merge, per style guide: https://github.com/openshift/openshift-docs/pull/69211#pullrequestreview-1791426688

Dropping merge labels for now but please re-queue when addressed. Thank you!

**Comment by mletalie** [comment]
@adellape, thanks for catching that...threw me off at first b/c I thought I had addressed it...which I did, but then made another change and re-added it! :( 
All set now.

**Comment by michaelryanpeter** [comment]
There is still an extra space before `any`, but you can feel free to clean that up in another pass. Not worth blocking a merge over.

LGTM

**Pull request merged**